pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -29,6 +29,9 @@ type ServiceBindingRequestSpec struct {
 	// BackingServiceSelector is used to identify the backing service operator.
 	BackingServiceSelector BackingServiceSelector `json:"backingServiceSelector"`
```

Comment:
Is this kept for backward compatibility?

Reply:
Yes; this PR is not meant to change functionality as of yet, just introduce some components and adjust some existing APIs they require so no user visible change should be introduced.

Reply:
+1